dev/com.ibm.ws.cdi.visibility_fat/.classpath

```diff
@@ -43,7 +43,7 @@
 	<classpathentry kind="src" path="test-applications/visTestEjbLib.jar/src"/>
 	<classpathentry kind="src" path="test-applications/maskedClassEjb.jar/src"/>
 	<classpathentry kind="src" path="test-applications/sharedLibraryNoInjectionApp.war/src"/>
-	<classpathentry kind="src" path="test-applications/warLibAccessBeansInWar.ear/src"/>
+	<classpathentry kind="src" path="test-applications/warLibAccessBeansInWar.ear/resources"/>
```

Comment:
Since there is no java source in this directory, I didn't think we included it in .classpath.  At least other projects that have 'resources' directories with just XML have not done this.